data-source/jsonnet/northern-ireland/census_household.jsonnet

```diff
@@ -1,31 +1,101 @@
+local disability_other = import 'individual/blocks/identity-and-health/disability_other.jsonnet';
+local ethnic_group = import 'individual/blocks/identity-and-health/ethnic_group.jsonnet';
+local frequency_irish = import 'individual/blocks/identity-and-health/frequency_irish.jsonnet';
+local frequency_ulster_scots = import 'individual/blocks/identity-and-health/frequency_ulster_scots.jsonnet';
+local health = import 'individual/blocks/identity-and-health/health.jsonnet';
+local language = import 'individual/blocks/identity-and-health/language.jsonnet';
+local last_year_address = import 'individual/blocks/identity-and-health/last_year_address.jsonnet';
+local national_identity = import 'individual/blocks/identity-and-health/national_identity.jsonnet';
+local no_religion = import 'individual/blocks/identity-and-health/no_religion.jsonnet';
+local passports = import 'individual/blocks/identity-and-health/passports.jsonnet';
+local past_usual_household_address = import 'individual/blocks/identity-and-health/past_usual_household_address.jsonnet';
+local religion = import 'individual/blocks/identity-and-health/religion.jsonnet';
+local sexual_identity = import 'individual/blocks/identity-and-health/sexual_identity.jsonnet';
+local speak_english = import 'individual/blocks/identity-and-health/speak_english.jsonnet';
+local understand_irish = import 'individual/blocks/identity-and-health/understand_irish.jsonnet';
+local understand_ulster_scots = import 'individual/blocks/identity-and-health/understand_ulster_scots.jsonnet';
+
+// School
+local school_location = import 'individual/blocks/school/school_location.jsonnet';
+local school_travel = import 'individual/blocks/school/school_travel.jsonnet';
+local study_location_type = import 'individual/blocks/school/study_location_type.jsonnet';
+
+// Qualifications
+local a_level = import 'individual/blocks/qualifications/a_level.jsonnet';
+local apprenticeship = import 'individual/blocks/qualifications/apprenticeship.jsonnet';
+local degree = import 'individual/blocks/qualifications/degree.jsonnet';
+local gcse = import 'individual/blocks/qualifications/gcse.jsonnet';
+local nvq_level = import 'individual/blocks/qualifications/nvq_level.jsonnet';
+local other_qualifications = import 'individual/blocks/qualifications/other_qualifications.jsonnet';
+local qualifications = import 'individual/blocks/qualifications/qualifications.jsonnet';
+
+// Employment
+local armed_forces = import 'individual/blocks/employment/armed_forces.jsonnet';
+local business_name = import 'individual/blocks/employment/business_name.jsonnet';
+local employer_address_depot = import 'individual/blocks/employment/employer_address_depot.jsonnet';
+local employer_address_workplace = import 'individual/blocks/employment/employer_address_workplace.jsonnet';
+local employer_type_of_address = import 'individual/blocks/employment/employer_type_of_address.jsonnet';
```

Comment:
These three files don't exist.

Reply:
Copy and pasted from the individual schema. I have removed this reference from the individual.jsonnet as well.